src/tests/ftest/dfuse/bash.py

```diff
@@ -1,5 +1,6 @@
 """
   (C) Copyright 2020-2024 Intel Corporation.
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

Comment:
The "(C)" characters have no legal meaning in Copyright notices at all, so should be dropped from these notices as they are maintained.

Copyright notices need to have the word "Copyright" *OR* the R in a circle copyright symbol and should not have both present in the notice, so even if the "(C)" had any meaning, it still should not be present.

The (C) should only be present in copyright notices that must match a boilerplate reference as it is likely that those boilerplate notices will not ever get fixed.

Reply:
Fixed in https://github.com/daos-stack/daos/pull/16224/commits/25461b16087c1177975e3126098aa88d1355c35c